packages/core/src/button/button.stories.ts

```diff
@@ -229,10 +229,10 @@ export function sizes() {
 export function block() {
   return html`
     <div cds-layout="vertical gap:sm align:horizontal-stretch">
-      <cds-button block>Default ('md')</cds-button>
-      <cds-button block action="outline">Default ('md')</cds-button>
-      <cds-button block size="sm">Compact ('sm')</cds-button>
-      <cds-button block action="outline" size="sm">Compact ('sm')</cds-button>
+      <cds-button>Default ('md')</cds-button>
+      <cds-button action="outline">Default ('md')</cds-button>
+      <cds-button size="sm">Compact ('sm')</cds-button>
+      <cds-button action="outline" size="sm">Compact ('sm')</cds-button>
```

Comment:
With the flex bug fix block is not needed as the button now behaves like native dom elements when placed as flex children.